protocol/dubbo/dubbo_invoker.go

```diff
@@ -36,16 +36,29 @@ import (
 
 var Err_No_Reply = perrors.New("request need @response")
 
+var (
+	attachmentKey = []string{constant.INTERFACE_KEY, constant.GROUP_KEY, constant.TOKEN_KEY, constant.TIMEOUT_KEY}
+)
+
 type DubboInvoker struct {
 	protocol.BaseInvoker
 	client      *Client
+	attachment  map[string]string
```

Comment:
you don't need to add this attachment in the `DubboInvoker`. if you want to set `attachmentKey `, I think you should do it in the `token_filter`. or here:
![image](https://user-images.githubusercontent.com/18375310/64909150-01844a00-d73b-11e9-8ef7-1b25827c9ff5.png)

Reply:
done